guide/src/migration.md

```diff
@@ -722,7 +722,7 @@ If `T` implements `Clone`, you can extract `T` itself.
 In addition, you can also extract `&PyCell<T>`, though you rarely need it.
 
 Before:
-```ignore
+```compile_fail
```

Comment:
I'm not sure it's worth changing bits like this which we know have old syntax, it's presumably just wasting CPU cycles to assert that the compile fails. Is there a particular intended benefit to this change?

Reply:
Well, I'd like to see to it that it doesn't start compiling again. I guess it doesn't matter because it's so old. IMO this one of these "don't waste time thinking about it, just do it" kind of things.

Reply:
Ok, though I might trim some of these back again another time if I ever try to optimise our CI pipeline. 👍